skbio/embedding/tests/test_embedding.py

```diff
@@ -69,6 +82,63 @@ def test_assert_length(self):
             SequenceEmbedding(self.emb, self.seq + "A")
 
 
+class SequenceVectorTests(TestCase):
+
+    def setUp(self):
+        # Create some sample SequenceVector objects for testing
+        self.vector1 = np.array([1, 2, 3])
+        self.vector2 = np.array([4, 5, 6])
+        self.vector3 = np.array([7, 8, 9])
+        self.sequence_vectors = [SequenceVector(self.vector1, "ACGT"),
+                                 SequenceVector(self.vector2, "GCTA"),
+                                 SequenceVector(self.vector3, "TTAG")]
+
+    def test_vector(self):
+        # Test if the vector attribute is set correctly
+        for i, vector in enumerate([self.vector1, self.vector2, self.vector3]):
+            self.assertTrue(np.array_equal(self.sequence_vectors[i].vector, vector))
+
+    def test_sequence(self):
+        # Test if the sequence attribute is set correctly
+        for i, sequence in enumerate(["ACGT", "GCTA", "TTAG"]):
+            self.assertEqual(self.sequence_vectors[i].sequence, sequence)
+
+    def test_repr(self):
+        # Test if the __repr__ method returns the correct string
+        for i, sequence_vector in enumerate(self.sequence_vectors):
+            self.assertTrue(
+                sequence_vector.__repr__().startswith("SequenceVector")
+            )
+
+    def test_str(self):
+        # Test if the __str__ method returns the correct string
+        for i, sequence_vector in enumerate(self.sequence_vectors):
+            self.assertEqual(str(sequence_vector), sequence_vector.sequence)
+
+    def test_to_numpy(self):
+        # Test if to_numpy returns the correct numpy array
+        expected_result = np.array([self.vector1, self.vector2, self.vector3])
+        result = SequenceVector.to_numpy(self.sequence_vectors)
+        self.assertTrue(np.array_equal(result, expected_result))
+
+    def test_to_distance_matrix(self):
+        # Test if to_distance_matrix returns a DistanceMatrix object
+        distance_matrix = SequenceVector.to_distance_matrix(self.sequence_vectors)
+        self.assertEqual(distance_matrix.shape, (3, 3))
```

Comment:
Testing the exact values present in the distance matrix should be easy and more precise of a test

Reply:
Same re: `to_dataframe` and `to_ordination`

Reply:
tests have been added